config-htsget-rs/download-config.toml

```
@@ -0,0 +1,24 @@
+ticket_server_addr = "0.0.0.0:8080"
+ticket_server_cors_allow_origins = "All"
+
+[[resolvers]]
+regex = "(.*)"
+substitution_string = "$1"
+
+[resolvers.object_type]
+send_encrypted_to_client = true  # the client will receive encrypted files
```

This gives the impression that HTSGET can decide if the data can be passed on encrypted or not.

---

Htsget needs to know if the coordinates should be for encrypted or unencrypted file

---

The name doesn't reflect that, the comment should maybe say that it related to the coordinates.

---

```suggestion
send_encrypted_to_client = true  # if true, htsget will calculate coordinates for the encrypted case
```

Valid point. How about my suggestion above?